Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(JSON: Dictionary<String, AnyObject>) {
```

Comment:
As @gfontenot mentioned it would be good to use Argo for the parsing which gives us a similar behavior for free. Since Argo *currently* relies on currying, I wonder if that's even an option right now due to the interop story?

For @tomkowz, what Argo would offer is that `dailyForecastWithJSON` method for free. And it would set us up in the future to use it for more of our JSON parsing as we need it.

I'm not sold on the hiding/showing the dailyForecasts, using the null-object pattern is something we do a lot because it removes logic like the one you defined above. Can you elaborate on why you don't like null-object?